tests/tests/optionsTests.robot

```diff
@@ -14,11 +14,14 @@ Test Setup          Prepare For Test
 Test Teardown       Clean After Test
```

Comment:
It does appear that these tests are not using __init__.robot file to prepare environment. I think you should merge changes from "tests" and delete these lines (unless these tests are using something different to prepare or clean)